pkgs/by-name/tu/turtle/package.nix

```
@@ -41,6 +41,9 @@ python3Packages.buildPythonApplication rec {
     dbus-python
   ];
 
+  # Upstream requiring pygit2==1.14.1
+  pythonRelaxDeps = [ "pygit2" ];
+
   postInstall = ''
     python ./install.py install
```

```suggestion
    ${python3Packages.python.interpreter} ./install.py install
```
or even `pythonOnBuildForHost.interpreter`.